src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -20,15 +28,37 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_TAG);
+        argMultimap.verifyOnlyOnePrefix();
+
+        return new FindCommand(createPredicate(argMultimap));
+    }
+
+    /**
+     * Creates the matching NetConnectPredicate based on the provided values in
+     * the given {@code argMultimap}.
+     *
+     * @throws ParseException if the given values are not valid
+     */
+    private static NetConnectPredicate<Person> createPredicate(
+            ArgumentMultimap argMultimap) throws ParseException {
+        if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
```

Comment:
Shall we handle the case where the user specify both prefix_name and prefix tag?

Currently, if this is the case, the code seems to just filter the search result by PREFIX_NAME.

Reply:
This case is handled in line 33 with `ArgumentMultimap#verifyOnlyOnePrefix`! Can be extended to when the find command accepts more arguments too.